Assignment5/src/controllers/asyncController.js

```diff
@@ -0,0 +1,13 @@
+const createError = require("http-errors");
+const promise = async (req, res, next) => {
```

Comment:
Promise is not correct as per naming convention. Please choose relevant names.

Reply:
Changes done